Packages/Primitives/Sources/Extensions/NSFileManager+Primitives.swift

```diff
@@ -3,6 +3,28 @@
 import Foundation
 
 public extension FileManager {
+    enum Directory: String {
+        case documents
+        case applicationSupport
+        case preferences
```

Comment:
```suggestion
        case library("Preferences")
```

maybe more consistent? and add `Preferences` to const